Content.Server/Zombies/ZombieSystem.Transform.cs

```diff
@@ -201,6 +207,11 @@ public void ZombifyEntity(EntityUid target, MobStateComponent? mobState = null)
 
         Dirty(target, melee);
 
+        // RMC14 - Give them night vision
+        var nightvisComp = EnsureComp<NightVisionComponent>(target);
+
+        Dirty(target, nightvisComp);
```

Comment:
```suggestion
        // RMC14 - Give them night vision
        EnsureComp<NightVisionComponent>(target);
```

you can shorten it to this, this variable isnt used anywhere

Reply:
done ty boss